GoogleDataTransportCCTSupport/GDTCCTTests/Unit/GDTCCTPrioritizerTest.m

```diff
@@ -168,9 +168,11 @@ - (void)testNetworkConnectionInfo {
   GDTCCTPrioritizer *prioritizer = [[GDTCCTPrioritizer alloc] init];
```

Comment:
Would you mind adding a test: `- (void)testEventCode` that does basic getting and setting?